core/azure-core/pom.xml

```diff
@@ -7,7 +7,7 @@
   <parent>
     <groupId>com.azure</groupId>
     <artifactId>azure-core-parent</artifactId>
-    <version>1.0.0-preview.1</version>
+    <version>1.0.0-preview.2</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
```

Comment:
The version property should be explicitly set in this pom.xml file as well. It is actually an issue blocking the release that it currently isn't there. 

FYI @scbedd

Reply:
Added the version.